### ISSUE TITLE
Add media controls and 'play on youtube' for trailers.

### ISSUE THREAD

**Issue by esran** [petio-team/petio#660]
**Is your feature request related to a problem? Please describe.**
I would like to see more controls on trailers. Volume adjustment, and option to pop out to youtube. Also the way they are shown it looks like the left hand edge is clipped.

**Describe the solution you'd like**
Instead of the heavily embedded view that looks like it clips the left edge and has no media controls, maybe have the whole youtube frame including controls (pause, volume, full screen, etc.) including the 'watch on youtube' button.

**Describe alternatives you've considered**
I mostly don't use the trailers feature due to the lack of media controls and the feel that it is clipped behind other UI elements.

**Additional context**
I think the trailer feature could be useful but needs these sort of enhancements so the user has more control and doesn't get blasted with something suddenly louder than they would like that they can't pause easily!

**Comment by AshDyson** [comment]
Yes this is planned in the UI rework currently under way. Trailers will be moved from embedded to modal style with full controls.

**Comment by marchershey** [comment]
> Trailers will be moved from embedded to modal style with full controls.

...but the way you have it now is sooo beautiful! Can you add controls to the existing design?

Also, on mobile, if the controls were there and only the exit button removed the player, that would be 10/10.

**Comment by AshDyson** [comment]
> > Trailers will be moved from embedded to modal style with full controls.
> 
> ...but the way you have it now is sooo beautiful! Can you add controls to the existing design?
> 
> Also, on mobile, if the controls were there and only the exit button removed the player, that would be 10/10.

We couldn't have the controls on mobile due to how the iframe interacted with mobile devices, I can't recall the exact issue but it was easier to just not have them.

I did like the way we have them now, but the new UI requires a different approach due to layout so it'll be shifting to a control modal.
